shared/variables.tf

```diff
@@ -0,0 +1,25 @@
+variable "region" {
+  default     = "ap-southeast-1"
```

Comment:
There was a good point raised on easyhotel previously on whether this needs to actually be dynamic.

Or whether we want to keep this as a local so we will have a code-change if we intend to change this. Then it might be better to have this as a local, similar to easyhotel

Reply:
Makes sense, I moved it to locals in [346e992](https://github.com/nvminhtue/tyson_devops_ic/pull/18/commits/346e992d89be49ee9dd223a249564b2c6684a74c)